Sources/MosaiqueAssetsPicker/Views/AssetDetailCell.swift

```diff
@@ -122,12 +127,18 @@ final class AssetDetailCell: UICollectionViewCell, AssetDetailCellBindable {
     }
 
     func setDownloading(_ isDownloading: Bool) {
-        if isDownloading, spinner == nil {
-            let spinner = UIActivityIndicatorView(style: .whiteLarge)
-            contentView.addSubview(spinner)
-            spinner.center = contentView.center
-            spinner.startAnimating()
-            self.spinner = spinner
+        if isDownloading {
+            let _spinner: UIActivityIndicatorView
```

Comment:
I find this pattern:

```
            let spinner = self.spinner ?? {
                let spinner = UIActivityIndicatorView(style: .whiteLarge)
                self.spinner = spinner
                contentView.addSubview(spinner)
                return spinner
            }()
```

More readable, and often make use of shadowing, but that's just a question of style!

Reply:
it's cool, let me use this pattern.